src/components/text/List/README.md

```diff
@@ -5,7 +5,7 @@ Basic unordered (default) list. When the list items are not 'li' elements, they
 	<a>Featured</a>
 	<a>New</a>
 	<a>Popular</a>
-	<a>made by Flywheel</a>
```

Comment:
I wonder if these need to be updated? Like the component was made by Flywheel right?